README.md

```diff
@@ -21,4 +131,111 @@ Things you may want to cover:
 
 * Deployment instructions
 
-* ...
+### Setup
+
+- Download the Zip file or Clone the repo with:
+   - git clone https://github.com/karam084/budget-app
+   - To access the project directory
+   - cd budget-app
+
+### Install
+
+ - bundle install
+ - rails db:migrate
+
+### Usage
+
+To run the project, execute the following command:
+
+- rails s 
+- Check http://localhost:3000/ in your browser
+
+### Deployment
+
+Run the following command to install all dependencies.
+
+$ bundle install
+$ code .
+
+## Database
+To create your local Database
+
+$ rails db:create
+$ rails db:migrate
+
+### Run tests
+To install rspec, in the terminal kindly run this command
+
+$ gem install rspec
+To run tests, please run this command
+
+$ rspec ./spec/#{filename}_spec.rb 
+To run all tests, please run this command
+
+or $ spec  
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Karam Fouad**
+- GitHub: [@karam084](https://github.com/karam084)
+- Twitter: [@ElarabFouad](https://twitter.com/ElarabFouad)
+- LinkedIn: [Karam Fouad](https://www.linkedin.com/in/karam-fouad-179830214/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Doplay App**
+- [ ] **Add live demo**
+- [ ] **Add Images**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+CContributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/karam084/budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Video Demo
+[loom video](https://www.loom.com/share/91412cdddc434ea99842e53da62a70a3)
+
+<!-- LICENSE -->
+
+## Design Template
+
+-  I give credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the author of the [original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken or is not set, kindly endeavor to fix it.